parlai/core/teachers.py

```diff
@@ -262,8 +265,7 @@ def __init__(self, opt, shared=None):
 
         # set up batching
         self.bsz = opt.get('batchsize', 1)
-        self.batchindex = opt.get('batchindex', 0)
-        self.use_batch_act = False  # Batch act disabled by default
+        self.use_batch_act = False
```

Comment:
Okay, don't immediately understand why we can get away with deleting .batch_act() altogether here. And we still have a flag for .use_batch_act even though the method has been deleted?

Reply:
can we just get rid of `use_batch_act` altogether now?